doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc

```diff
@@ -0,0 +1,361 @@
+* `db`: Database; requires additional configuration in the `jbpm.usergroup.callback.properties` file.
+* `jaas`: JAAS; delegates to the container to fetch information about user data.
+* `props`: A simple property file; requires additional file that will keep all information (users and groups).
+* `custom`: A custom implementation; specify the fully qualified name of the class in the `org.jbpm.ht.custom.callback` property.
+
+|`org.jbpm.ht.custom.callback`
+|Fully qualified name
+|N/A
+|A custom implementation of the `UserGroupCallback` interface in case the `org.jbpm.ht.callback` property is set to `custom`.
+
+|`org.jbpm.task.cleanup.enabled`
+|`true`, `false`
+|`true`
+|Enables task cleanup job listener to remove tasks once the process instance is completed.
+
+|`org.jbpm.task.bam.enabled`
+|`true`, `false`
+|`true`
+|Enables task BAM module to store task related information.
+
+|`org.jbpm.ht.admin.user`
+|String
+|`Administrator`
+|User who can access all the tasks from {KIE_SERVER}.
+
+|`org.jbpm.ht.admin.group`
+|String
+|`Administrators`
+|In order to display all the tasks from {KIE_SERVER}, users must belong to the `Administrators` group.
+|===
+endif::[]
+
+.System properties for loading keystore
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`kie.keystore.keyStoreURL`
+|URL
+|N/A
+|URL is used to load a Java Cryptography Extension KeyStore (JCEKS). For example, `file:///home/kie/keystores/keystore.jceks`.
+
+|`kie.keystore.keyStorePwd`
+|String
+|N/A
+|The password is used for the JCEKS.
+
+|`kie.keystore.key.server.alias`
+|String
+|N/A
+|The alias name of the key for REST services where the password is stored.
+
+|`kie.keystore.key.server.pwd`
+|String
+|N/A
+|The password of an alias for REST services.
+
+|`kie.keystore.key.ctrl.alias`
+|String
+|N/A
+|The alias of the key for default REST {CONTROLLER}.
+
+|`kie.keystore.key.ctrl.pwd`
+|String
+|N/A
+|The password of an alias for default REST {CONTROLLER}.
+|===
+
+.Other system properties
+[cols="2,1,1,3a", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`kie.maven.settings.custom`
+|Path
+|N/A
+|The location of a custom `settings.xml` file for Maven configuration.
+
+|`kie.server.jms.queues.response`
+|String
+|`queue/KIE.SERVER.RESPONSE`
+|The response queue JNDI name for JMS.
+
+|`org.drools.server.filter.classes`
+|`true`, `false`
+|`false`
+|When set to `true`, the Drools {KIE_SERVER} extension accepts custom classes annotated by the `XmlRootElement` or `Remotable` annotations only.
+
+ifdef::PAM,JBPM[]
+|`org.kie.server.bypass.auth.user`
+|`true`, `false`
+|`false`
+|A property that enables you to bypass the authenticated user for task-related operations, for example queries.
+
+|`org.jbpm.rule.task.firelimit`
+|Integer
+|`10000`
+|A property that specifies maximum number of fired rules to avoid situation where rules run into infinite loop and make server completely unresponsive.
```

Comment:
Should be a complete sentence, and also best to avoid "fire" which is jargon, and use "execute" instead. Suggest "... specifies the maximum number of executed rules to avoid situations where rules run into an infinite loop and make the server completely unresponsive"